sticker/Cargo.toml

```diff
@@ -14,7 +14,7 @@ license-file = "../LICENSE.md"
 [dependencies]
 conllx = "0.11"
 failure = "0.1"
-finalfusion = "0.9"
+finalfusion = "0.10.1"
```

Comment:
Is there motivation to fully specify the version here while leaving it at `0.10` for `sticker-utils`?

Reply:
Only 0.10.1 has the `quantizer` getter, so we need that as a lower bound.